content/introduction/_index.md

```diff
@@ -1,60 +1,187 @@
 ---
-title: Introduction
+title: "Introduction"
 weight: 1
 pre: '<i class="fas fa-fw fa-comments"></i> '
 ---
 
-Welcome to the libp2p documentation portal! Whether you’re just learning how to build peer-to-peer systems with libp2p, want to dive into peer-to-peer concepts and solutions, or are looking for detailed reference information, this is the place to start.
+Welcome to the official libp2p documentation hub!
 
-## Overview
+Whether you’re just starting to dive into peer-to-peer concepts and 
+solutions, learning how to build peer-to-peer systems with libp2p, or 
+are looking for detailed reference information, this is the place to 
+start.
 
-Head over to [What is libp2p?](/introduction/what-is-libp2p/) for an introduction to the basics of libp2p and an overview of the problems it addresses.
+## What is libp2p?
 
+libp2p is a modular system of *protocols*, *specifications* and *libraries* 
+that enable the development of peer-to-peer network applications. 
 
-## Tutorials
+**The networking protocol of web3**
 
-If you want to dive in, check out our collection of [tutorials](/tutorials/), which will help guide you through your explorations of libp2p.
+Because of libp2p's peer-to-peer and distributed architecture, most of the 
+needs and considerations that the current web was built on no longer apply.
+Moving from a location-addressed system to a peer-to-peer, content-addressed 
+system presents many challenges. The internet, such as it is, with firewalls 
+and NATs, was designed to [securely] provide data by relying on trust assumptions, 
+Clients must trust the service providers they connect to and the data they provide, 
+despite servers needing to verify their identity using certificate authorities.
```

Comment:
This seems like a motivation for IPFS, not libp2p.

Reply:
Right, updated.